src/pdf.rs

```diff
@@ -281,11 +281,13 @@ mod test {
 
     #[test]
     fn writer() {
-        let file = ::std::fs::File::create("test1.pdf").unwrap();
+        let filename = "test_x.pdf";
+        let file = ::std::fs::File::create(filename).unwrap();
         let surface = Writer::new(100., 100., file);
 
         draw(&surface);
         surface.finish();
+        ::std::fs::remove_file(filename).unwrap();
```

Comment:
In a case of a panic, will the drop be called? I'm not so sure about it...

Reply:
Yes, that's the point :) I'm sure there are temporary file crates that implement this already.

Please change that :)

Reply:
In `glib` we use the `tempdir` crate for the same purpose, but on directories. You could also use that here to create a temporary directory and then a file in there, and it will automatically be removed once it is dropped (including panics).

Reply:
I will. I merged the PR so I will open an issue about it.